backend/modules/pdf_processing/ai_extraction_service.py

```
@@ -168,9 +168,9 @@ async def extract_personas_and_key_figures(
 CRITICAL: You must identify ALL named individuals, companies, organizations, and significant unnamed roles mentioned within the case study narrative.
 
 ━━━ KEY FIGURES IDENTIFICATION ━━━
-- Find ALL key figures that can be turned into simulation personas (NPCs)
-- Include both named and unnamed roles that appear in the business story
-- Even briefly-mentioned figures should be included if they have a clear role
+- Find ALL named individuals and significant roles — aim for at least 4–6 personas
```

Ruff (0.15.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)